I got 1 error:
```
================================================================ FAILURES ================================================================
______________________________________________________________ test_tyk_api ______________________________________________________________

    def test_tyk_api():
        if KEYCLOAK_PUBLIC_URL is None:
            warnings.warn(UserWarning("KEYCLOAK_URL is not set"))
            return
    
        token = authx.auth.get_access_token(
        keycloak_url=KEYCLOAK_PUBLIC_URL,
        username=SITE_ADMIN_USER,
        password=SITE_ADMIN_PASSWORD
        )
        policy_id="testtest"
        response = authx.auth.add_provider_to_tyk_api("91", token, f"{KEYCLOAK_PUBLIC_URL}/auth/realms/candig", policy_id=policy_id)
>       assert response.status_code == 200
E       assert 403 == 200
E        +  where 403 = <Response [403]>.status_code

test_auth.py:238: AssertionError
======================================================== short test summary info =========================================================
FAILED test_auth.py::test_tyk_api - assert 403 == 200
====================================================== 1 failed, 9 passed in 3.79s =======================================================
```
test-integration are still passing so I think tyk is working properly, not sure what causing this

--- a/test_auth.py
+++ b/test_auth.py
@@ -236,7 +236,7 @@ def test_tyk_api():
     policy_id="testtest"
     response = authx.auth.add_provider_to_tyk_api("91", token, f"{KEYCLOAK_PUBLIC_URL}/auth/realms/candig", policy_id=policy_id)
     assert response.status_code == 200
-    time.sleep(1) # tyk takes a second to refresh this after reloading
+    time.sleep(5) # tyk takes a second to refresh this after reloading
     url = f"{TYK_LOGIN_TARGET_URL}/tyk/apis/91"
     headers = { "x-tyk-authorization": TYK_SECRET_KEY }
     response = requests.request("GET", url, headers=headers)
@@ -251,11 +251,11 @@ def test_tyk_api():
     count = len(response.json()['openid_options']['providers'])
     response = authx.auth.add_provider_to_tyk_api("91", token, f"{KEYCLOAK_PUBLIC_URL}/auth/realms/candig", policy_id=policy_id)
     assert response.status_code == 200
-    time.sleep(1) # tyk takes a second to refresh this after reloading
+    time.sleep(5) # tyk takes a second to refresh this after reloading
     assert len(response.json()['openid_options']['providers']) == count
 
     response = authx.auth.remove_provider_from_tyk_api("91", KEYCLOAK_PUBLIC_URL, policy_id=policy_id)
-    time.sleep(1) # tyk takes a second to refresh this after reloading
+    time.sleep(5) # tyk takes a second to refresh this after reloading
     assert response.status_code == 200
     response = requests.request("GET", url, headers=headers)
     print(response.json()['openid_options']['providers'])

